tidalapi/session.py

```diff
@@ -465,6 +465,12 @@ def token_refresh(self, refresh_token):
         self.token_type = json["token_type"]
         return True
 
+    def set_audio_quality(self, quality=media.Quality.high):
```

Comment:
Why not properties?

```python
@property
def audio_quality(self) -> media.Quality:
    return self.config.quality # or validate via the enum
    
 @audio_quality.setter
 def audio_quality(self, quality=...
 ```

Reply:
You are right, I should get familiar with python decorators - they are neat!

So something like this?
```python
@property
    def audio_quality(self) -> media.Quality:
        return self.config.quality

    @audio_quality.setter
    def audio_quality(self, value=media.Quality.high):
        self.config.quality = media.Quality(value)
```
and
```
print(self._active_session.audio_quality)
self._active_session.audio_quality = Quality.master
```

Reply:
Nearly: you can't have the default on the setter though.  I'll suggest tmrw